### DISCUSSION TITLE
Search box: send query to a new page (fragment url)

### DISCUSSION THREAD

**Discussion by ileana2019** [microsoft-search/pnp-modern-search#1949] [Q&A]
Hello,
I have a question: On a SharePoint site Home page I have a search box configured to send the query to a new page, Query input transformation template: {inputQueryText}, Method: URL fradment. When searching for something in the search box on the home page, I will get redirected to the url page I specified in the webpart properties, having in the url the fragment with hashtag. On the redirected page, I have again a search box and search results webparts, connected to each other. How should I configure the query template of the search results webpart in the redirect page to search for the hashtag fragment in the url?
Thank you
![Search Box-Home Page](https://user-images.githubusercontent.com/55545989/158833812-14d556f9-4564-4324-9ad9-93f25a6bfebd.jpg)
!
![Search Results-Redirect Page](https://user-images.githubusercontent.com/55545989/158833833-c7d1aabc-fd19-49b4-9b86-350464f333c8.PNG)
![Search Box-Redirect Page](https://user-images.githubusercontent.com/55545989/158833854-23188c36-8dde-4819-add7-d6f097a9aaf0.PNG)

**Comment by nicolesherman00** [comment]
I am sad this didnt have any answers. I am here having trouble with the same thing.

**Comment by wobba** [comment]
Configure it like this:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/51104/4b09ca2d-e271-4668-89ec-3a877b73d3a6)
